### PR TITLE
fix: fix listevents docstring parameter mismatch

### PR DESCRIPTION
**Pull request by alisterburt**

### CODE REVIEW

File: src/psygnal/containers/_evented_list.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def __setitem__(self, key: Index, value: Union[_T, Iterable[_T]]) -> None:
                 raise TypeError("Can only assign an iterable to slice")
             value = [self._pre_insert(v) for v in value]  # before we mutate the list
         else:
-            value = self._pre_insert(cast(_T, value))
+            value = self._pre_insert(cast("_T", value))
```
Comment (alisterburt) [inline src/psygnal/containers/_evented_list.py:173]: pre-commit was complaining about this

**Comment by tlambert03** [comment]
oh my... looks like we've got some other issues too :/  will take a look.  thanks!

**Comment by alisterburt** [comment]
Sorry to bring problems to the forefront!

File: src/psygnal/containers/_evented_list.py [hunk 2]
```diff
@@ -58,8 +58,8 @@ class ListEvents(SignalGroup):
         `(index, value)` emitted after `value` is removed at `index`
     moving : Signal[int, int]
         `(index, new_index)` emitted before an item is moved from `index` to `new_index`
-    moved : Signal[int, int]
-        `(index, new_index, value)` emitted after `value` is moved from `index` to
+    moved : Signal[tuple, int]
```
Comment (tlambert03) [inline src/psygnal/containers/_evented_list.py:61]: actually it looks like it should be `Signal[int, int, int]` ... see line 283, where the actual emit happens

**Pull request merged**